system/jlib/jstatcodes.h

```diff
@@ -69,6 +69,7 @@ enum StatisticScopeType
     SSTfunction,                        // a function call
     SSTworkflow,
     SSTchildgraph,
+    SSTnote,
```

Comment:
It isn't really a scope type - it is a type of property.  It doesn't exist independently of a scope.